pom.xml

```diff
@@ -108,6 +108,11 @@
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_2.11</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.jsuereth</groupId>
+            <artifactId>scala-arm_2.11</artifactId>
+            <version>2.0.0-M1</version>
```

Comment:
`version` not necessary, is inherited from parent pom. By the way, please upgrade parent pom to version `1.31`.